lib/mshoplib/src/MShop/Common/Item/Helper/Form/Standard.php

```diff
@@ -33,15 +34,17 @@ class Standard implements \Aimeos\MShop\Common\Item\Helper\Form\Iface
 	 * @param string $method Initial method (e.g. post or get)
 	 * @param array $values Form parameters implementing \Aimeos\MW\Criteria\Attribute\Iface
 	 * @param boolean $external True if URL points to an external site, false if it stays on the same site
+	 * @param string $html Custom HTML for rendering form (e.g. Including JS or custom html for Stripe Elements)
```

Comment:
Can you please remove "for Stripe Elements" as it is not specific for that

Reply:
Removed :-)